alacritty_terminal/src/term/mod.rs

```diff
@@ -1293,9 +1322,9 @@ impl<T> Term<T> {
             !self.mode.contains(TermMode::SHOW_CURSOR) || point.line >= self.screen_lines();
         let cursor_style = if hidden && !vi_mode {
             point.line = Line(0);
-            CursorStyle::Hidden
+            CursorStyle { shape: CursorShape::Hidden, blinking: false }
         } else if !self.is_focused && config.cursor.unfocused_hollow() {
-            CursorStyle::HollowBlock
+            CursorStyle { shape: CursorShape::HollowBlock, blinking: false }
```

Comment:
The unfocused hollow block cursor never blinks, but it seems like other cursors still blink even without focus.

I think if Alacritty isn't focused, the cursor shouldn't blink. I wouldn't even make this configurable, since it's just a waste of resources really.

Reply:
Indeed, disabling blinking while unfocused also is the behavior of XTerm and rxvt, I'll try to make it work too.